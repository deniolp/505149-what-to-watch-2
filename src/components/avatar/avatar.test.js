import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import Avatar from './avatar';

describe(`Avatar`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <BrowserRouter><Avatar
          user={{
            id: 1,
            name: `Olga`,
            email: `r@ya.ru`,
            avatarUrl: `/img/photo.jpg`,
          }}
        /></BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
