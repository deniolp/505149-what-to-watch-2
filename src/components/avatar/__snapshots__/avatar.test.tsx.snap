// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <a
      href="/mylist"
      onClick={[Function]}
      style={
        Object {
          "color": "#dfcf77",
          "textDecoration": "none",
        }
      }
    >
      <img
        alt="User avatar"
        height="63"
        src="/img/photo.jpg"
        width="63"
      />
    </a>
  </div>
</div>
`;
